nsca_client/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2015 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "NSCA Client",
+    "summary": "Send passive alerts to monitor your Odoo application.",
+    "version": "8.0.1.0.0",
+    "category": "Tools",
+    "website": "http://osiell.com/",
+    "author": "ABF OSIELL, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
+        "bin": ['/usr/sbin/send_nsca'],
```

Comment:
@StefanRijnhart it is just a format problem not a technical one.
My point was suggesting that for all other keys the brakets are in one separate line and we should be consistent in this one.
No big deal